public/kbn_sankey_vis_controller.js

```diff
@@ -149,12 +147,15 @@ module.controller('KbnSankeyVisController', function ($scope, $element, $rootSco
       link.attr('d', path);
     }
   };
-  var _render = window.render = function (data) {
+  let _render = window.render = function (data) {
     d3.select(svgRoot).selectAll('svg').remove();
     _buildVis(data);
   };
-  var data;
+  let data;
   $scope.$watch('esResponse', function (resp) {
+    console.log(resp);
```

Comment:
Hi @ch-bas , are these lines really intended to stay here?

Reply:
Hi @numbnut , sorry my mistake!